controls/V-71975.rb

```diff
@@ -31,80 +12,79 @@
-AIDE to scan for changes to the system baseline. The commands used in the example
-will use a daily occurrence.
+Check for the presence of a cron job running routinely on the system that
+executes AIDE to scan for changes to the system baseline. The commands used in
+the example will use a daily occurrence.
 
-Check the \"/etc/cron.daily\" subdirectory for a \"crontab\" file controlling the
-execution of the file integrity application. For example, if AIDE is installed on
-the system, use the following commands:
+Check the \"/etc/cron.daily\" subdirectory for a \"crontab\" file controlling
+the execution of the file integrity application. For example, if AIDE is
+installed on the system, use the following commands:
 
 # ls -al /etc/cron.daily | grep aide
 -rwxr-xr-x  1 root root        32 Jul  1  2011 aide
 
-AIDE does not have a configuration that will send a notification, so the cron job
-uses the mail application on the system to email the results of the file integrity
-run as in the following example:
+AIDE does not have a configuration that will send a notification, so the cron
+job uses the mail application on the system to email the results of the file
+integrity run as in the following example:
 
 # more /etc/cron.daily/aide
-0 0 * * * /usr/sbin/aide --check | /bin/mail -s \"$HOSTNAME - Daily aide integrity
-check run\" root@sysname.mil
+0 0 * * * /usr/sbin/aide --check | /bin/mail -s \"$HOSTNAME - Daily aide integrity check run\" root@sysname.mil
 
-If the file integrity application does not notify designated personnel of changes,
-this is a finding."
+If the file integrity application does not notify designated personnel of
+changes, this is a finding."
   tag "fix": "Configure the operating system to notify designated personnel if
-baseline configurations are changed in an unauthorized manner. The AIDE tool can be
-configured to email designated personnel through the use of the cron system.
+baseline configurations are changed in an unauthorized manner. The AIDE tool
+can be configured to email designated personnel through the use of the cron
+system.
 
-The following example output is generic. It will set cron to run AIDE daily and to
-send email at the completion of the analysis.
+The following example output is generic. It will set cron to run AIDE daily and
+to send email at the completion of the analysis.
 
 # more /etc/cron.daily/aide
-0 0 * * * /usr/sbin/aide --check | /bin/mail -s \"$HOSTNAME - Daily aide integrity
-check run\" root@sysname.mil"
-
-  describe package(file_integ_tool) do
+0 0 * * * /usr/sbin/aide --check | /bin/mail -s \"$HOSTNAME - Daily aide integrity check run\" root@sysname.mil"
+  tag "fix_id": "F-78327r1_fix"
+  describe package(FILE_INTEGRITY_TOOL) do
     it { should be_installed }
   end
   describe.one do
-    describe file("cat /etc/cron.daily/#{file_integ_tool}") do
-      its('content') { should match /\/bin\/mail/ }
+    describe file("/etc/cron.daily/#{FILE_INTEGRITY_TOOL}") do
```

Comment:
This could also be in root's crontab or `/etc/cron.d` so those should also be checked.